src/mono/wasm/features.md

```diff
@@ -368,6 +368,14 @@ You can add following elements in your .csproj
 </PropertyGroup>
 ```
 
+### Debugging non-debug configuration
+If you want to debug the application in other than Debug configuration, you need to set MSBuild property `DebuggerSupport` to `true` (globally or with condition for that particular configuration).
+```xml
+<PropertyGroup Condition="'$(Configuration)' == '[DESIRED CONFIGURATION NAME]'">
+  <DebuggerSupport>true</DebuggerSupport>
+</PropertyGroup>
+```
```

Comment:
please describe the costs

Reply:
Like
```suggestion
```
Debugging in `Release` mode can affect the behavior of your code due to optimizations, making it harder to trace the execution flow, observe variable values, or set breakpoints.
```
?